src/sinol_make/commands/run/__init__.py

```diff
@@ -594,8 +682,12 @@ def warn_if_not_empty(set, message):
         warn_if_not_empty(diff.removed_groups, "Groups were removed")
 
         for change in diff.changes:
-            print(util.warning("Solution %s passed group %d with status %s while it should pass with status %s." %
-                               (change.solution, change.group, change.result, change.old_result)))
+            if isinstance(change.result, str):
+                print(util.warning("Solution %s passed group %d with status %s while it should pass with status %s." %
+                                   (change.solution, change.group, change.result, change.old_result)))
+            elif isinstance(change.result, int):
+                print(util.warning("Solution %s passed group %d with %d points while it should pass with %d points") %
+                      (change.solution, change.group, change.result, change.old_result))
```

Comment:
The indentation here is different than three lines above.

Reply:
Fixed 004c72c88908b8b72be562269013c9a31d999554